_posts/2018-04-19-hash-authentication-test.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Hash Authentication Test
+weight: 5
+category: channel-api
+categoryItemType: documentation
+categoryItemIsShown: 1
+categoryItemWidth: 6
+categoryItemDescription: Test the API response for your Secret Key and Authentication Hash JSON.
+categoryItemLabel: Run an API test
+---
+<article data-sidenav="sidenav">
+	<section>
+
+		<h2>Test your authentication</h2>
+		
+		<p>On this page you can test the API response for your API Secret Key and the JSON token containing the authentication hash. Learn how to obtain these in the <a href="/channel-api/viewer-authentication-api.html">Viewer Authentication API document</a>.</p>
+		
+		<div class="form__control-group">
+			<label for="api-secret">Your API Secret Key</label>
```

Comment:
No need API secret key for hash. Hash generated via a secret which is set for the channel: http://developers.video.ibm.com/channel-api/viewer-authentication-api.html#setting-up-viewer-authentication_2